### ISSUE TITLE
Action fails on Windows and MacOS

### ISSUE THREAD

**Issue by evandrocoan** [axel-op/googlejavaformat-action#4]
I had setup it to run on Linux/Window/Mac OS, but one of them if failing this that error:
1. https://github.com/evandroforks/AppointmentPublishing/runs/903562824?check_suite_focus=true#step:4:11

![image](https://user-images.githubusercontent.com/5332158/88315557-ceacca00-ccec-11ea-85a5-fb187f01801e.png)

Only now I realiaze I can only run it in one of the machines, otherwise, I would get double commits.

The correct usage would be:
```yaml
jobs:

  test:

    name: Tests on ${{ matrix.os }} JDK ${{ matrix.java }}

    runs-on: ${{ matrix.os }}

    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
        java: [1.8, 1.11]

      # Keep all systems running if something fails
      fail-fast: false

    steps:
      - uses: actions/checkout@v2

      - name: Set up JDK
        uses: actions/setup-java@v1
        with:
          java-version: ${{ matrix.java }}

      # Automatically format and commit code with Google style guide
      # https://github.com/axel-op/googlejavaformat-action/issues/4
      - uses: axel-op/googlejavaformat-action@v3
        if: matrix.java == '1.11' && matrix.os == 'ubuntu-latest'
        with:
          args: "--skip-sorting-imports --replace"
```

**Comment by axel-op** [comment]
That's because I've only built this action for Linux machines, my bad. I'll try to make this cross-platform in a future version.

Thanks for the issue!

**Comment by axel-op** [comment]
Here's why this action fails on non-Linux machines:

- On MacOS: calls to GitHub API are rejected when trying to get the releases of GJF (related to [this issue](https://github.com/actions/virtual-environments/issues/602)). The solution would be to add a require parameter to this action for the GITHUB_TOKEN.
- On Windows: the [`HOME` environment variable](https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables) is not set and I cannot figure out why.

**Comment by axel-op** [comment]
I managed to find a workaround on Windows, so this action will now work on this OS.

**Comment by axel-op** [comment]
Issue "fixed" (with a workaround) with `v3.3.0`.
